src/pybind/mgr/mgr_module.py

```diff
@@ -377,11 +398,19 @@ def load_func_metadata(f: HandlerFuncType) -> Tuple[str, Dict[str, Any], int, st
                                                dict(name=arg),
                                                has_default,
                                                positional))
+        for argname, argtype in extra_args.items():
+            # avoid shadowing args from the function
+            if argname in arg_spec:
+                continue
+            arg_spec[argname] = argtype
+            args.append(CephArgtype.to_argdesc(
+                argtype, dict(name=arg), has_default=True, positional=False
+            ))
         return desc, arg_spec, first_default, ' '.join(args)
 
     def store_func_metadata(self, f: HandlerFuncType) -> None:
         self.desc, self.arg_spec, self.first_default, self.args = \
-            self.load_func_metadata(f)
+            self._load_func_metadata(f)
 
```

Comment:
Why not making more evident that this PR is now adding some specific processing for the `--format` flag? It takes a thorough review to find where the `format` flag is actually been processed.

Reply:
Let me see if I can add some more doctext to make it clearer.